ga4gh/cli.py

```diff
@@ -1185,7 +1205,8 @@ def __init__(self, args):
         super(DestroyRunner, self).__init__(args)
 
     def run(self):
-        self.repoManager.destroy()
+        def func(): self.repoManager.destroy()
+        self.confirmRun(func, 'the Data Repository')
```

Comment:
Might as well just have ``self.confirmRun(self.repoManager.destroy, 'the Data Repository')``, right?